api/followData.ts

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+
+interface User {
+  login: string;
+  avatar_url: string;
+}
+
+export const fetchNonFollowingUsers = async (
+  token: string,
+): Promise<{ following: User[]; nonFollowing: User[]; matchingUsers: User[] }> => {
+  const BASE_URL = 'https://api.github.com';
+  const PER_PAGE = 100;
+
+  const followingResponse = await axios.get<User[]>(`${BASE_URL}/user/following?per_page=${PER_PAGE}`, {
+    headers: {
+      Authorization: `token ${token}`,
+    },
+  });
+
+  const following = followingResponse.data;
```

Comment:
```suggestion
  const {data: following}= await axios.get<User[]>(`${BASE_URL}/user/following?per_page=${PER_PAGE}`, {
    headers: {
      Authorization: `token ${token}`,
    },
  });
```

요로케도 더 깔끔하게 가눙할듯 !!!